plugins/woocommerce/client/admin/client/settings-payments/settings-payments-offline.tsx

```diff
@@ -12,14 +14,88 @@ import './settings-payments-offline.scss';
 import { OfflinePaymentGateways } from './components/offline-payment-gateways';
 
 export const SettingsPaymentsOffline = () => {
-	const registeredPaymentGateways = useSelect( ( select ) => {
-		return select( PAYMENT_GATEWAYS_STORE_NAME ).getPaymentGateways();
-	}, [] );
+	const [ isEnabled, setIsEnabled ] = useState< boolean >( false );
+
+	// Make UI to refresh when plugin is installed.
+	const { invalidateResolution } = useDispatch( 'core/data' );
+
+	useEffect( () => {
+		if ( isEnabled ) {
+			invalidateResolution(
+				PAYMENT_GATEWAYS_SUGGESTIONS_STORE_NAME,
+				'getPaymentGateways'
+			);
+		}
+	}, [ isEnabled, invalidateResolution ] );
+
+	const { offlinePaymentGateways, isFetching } = useSelect( ( select ) => {
+		return {
+			registeredPaymentGateways: select(
```

Comment:
I don't use it, but I need to pull it to make the request to the store. I believe there's a better way to introduce getOfflineGateways action, right? cc @woocommerce/ghidorah

Reply:
I've created separate resolver and actions for that + modified reducer in https://github.com/woocommerce/woocommerce/pull/52824/commits/235fc200ad15ecdfdf7c46ebac3d1115cd4c4274.

Reply:
I left a comment suggesting a better way for this. I think we can remove the duplicate API call and offline actions by reusing `getPaymentGateways`. 🙂